### DISCUSSION TITLE
Some people think crouton is dead

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#5085] [General]
Here is a direct quote from a user on Reddit:

Crouton has been discontinued. It's not worth fiddling with for 99 percent of people.

It's so strange, in a way. It's almost like crouton has become politicized. Sure, it's a little trickier to set up than it used to be. But it's well worth it, to me.

Thanks, @dnschneid, for continuing to support it. Here is a link to my recent Reddit post: 

[Crouton has not been abandoned](https://www.reddit.com/r/Crouton/s/jPvu86z5Zn)

**Comment by mangolover1899** [comment]
Link to such reddit post?

**Reply by CroutonIsFun** [reply]
[Here is Reddit post](https://www.reddit.com/r/Crouton/comments/1dt1iqc/crouton_has_not_been_abandoned/)
